charts/testkube-api/templates/role.yaml

```diff
@@ -1,7 +1,8 @@
+{{ if .Values.rbac.createRoles }}
 apiVersion: {{ include "global.capabilities.rbac.apiVersion" . }}
-kind: ClusterRole
+kind: Role
 metadata:
-  name: executors-cluster-role-{{ .Release.Name }}
+  name: executors-role-{{ .Release.Name }}-{{ .Release.Namespace }}
```

Comment:
do we need the `-{{ .Release.Namespace }}` suffix now that is is namespace-scoped?

Reply:
I will remove it once fix for namespaces-scoped role is released. Now it's been added for testing purposes